src/Tinker.jl

```diff
@@ -12,24 +12,40 @@ struct Rectangle <: Shape
     y::Number
     w::Number
     h::Number
+    pts::AbstractArray
```

Comment:
Oh, yes: that's a product of generalizing my draw function to be able to draw any type of selection. Now everything is drawn from an array of coordinates. Perhaps I should calculate this later, though.